webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -280,4 +278,28 @@ test.describe('Experiment List', () => {
     // });
     // await test.step('Hyperparameter Search', async () => {});
   });
+
+  test('DataGrid Action Pause', async () => {
+    // create paused experiment
+    detExecSync(
+      `experiment create ${fullPath('examples/tutorials/mnist_pytorch/adaptive.yaml')}  --paused`,
+    );
+    await waitTableStable();
+
+    // experiment should initially be paused
+    const row = projectDetailsPage.f_experimentList.dataGrid.getRowByIndex(0);
+    await expect.soft((await row.getCellByColumnName('State')).pwLocator).toHaveText('paused');
+
+    // resume experiment
+    await row.experimentActionDropdown.open();
+    await waitTableStable();
```

Comment:
no need to wait table stable when opening the dropdown. you could probably just remove these lines